docker-compose-gpu.yml

```diff
@@ -24,6 +24,7 @@ services:
       # See rest_api/pipelines.yaml for configurations of Search & Indexing Pipeline.
       - DOCUMENTSTORE_PARAMS_HOST=elasticsearch
       - PIPELINE_YAML_PATH=/home/user/rest_api/pipeline/pipelines_dpr.yaml
+      - CONCURRENT_REQUEST_PER_WORKER=12
     depends_on:
       - elasticsearch
     command: "/bin/bash -c 'sleep 10 && gunicorn rest_api.application:app -b 0.0.0.0 -k uvicorn.workers.UvicornWorker --workers 1 --timeout 180'"
```

Comment:
You should be able to increase the number of workers here via the `--workers` arg for gunicorn. Did you try this?

Reply:
I thought that was a totally different kind of worker but actually I didn't check :sweat_smile: I'm going to remove the precise number from here, so I can play with it without enforcing a default value over the existing one.